test/relayserver/ServerTestUtils.ts

```diff
@@ -0,0 +1,86 @@
+import { TransactionReceipt } from '@ethersproject/providers';
+import {
+  RelayHubInterface,
+  RelayHub__factory,
+} from '@rsksmart/rif-relay-contracts';
+import { ManagerEvent } from '@rsksmart/rif-relay-server';
+import { BigNumberish, constants } from 'ethers';
+import { expect } from 'chai';
+import { ethers as hardhat } from 'hardhat';
```

Comment:
why do we need to rename it? It's the same object in runtime as directly from ethers library with added helper functions.

Reply:
I just followed the same structure that was usted in the relay-contracts, buy I could keep the same name.

Reply:
> not a big deal, but why do we need to rename it? It's the same object in runtime as directly from ethers library with added helper functions.

I agree that unless strictly necessary, I'd avoid renaming, because I find it a bit confusing.

Reply:
Ill update it